### PR TITLE
Add scripts for setting up hdfs

### PR DESCRIPTION
**Pull request by wkk**
Usage: `python3 deploy-hdfs.py config.json`

### CODE REVIEW

**Comment by wkk** [comment]
resolves  #732

File: bin/README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ on a remoate cluster when Hillview is installed
 
 * `delete-data.py`: delete a folder from all machines in a Hillview cluster
 * `deploy.py`: copy the Hillview binaries to all machines in a Hillview cluster
+* `deploy-hdfs.py`: download and setup HDFS to machines in a hillview cluster
```
Comment (mbudiu-vmw) [inline bin/README.md:56]: download and install HDFS on all ...

File: bin/deploy-hdfs.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# requires ansible and ansible-runner: pip3 install --user ansible ansible-runner
+import os
+from argparse import ArgumentParser
+from configparser import ConfigParser
+from tempfile import NamedTemporaryFile
+
+import ansible_runner
```
Comment (mbudiu-vmw) [inline bin/deploy-hdfs.py:8]: Can you check whether the import fails and then give an error containing the first line in this file?

File: bin/deploy-hdfs.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    inventory.add_section(DEFAULT_VARS)
+    inventory.set(DEFAULT_VARS, "ansible_user", config.get_user())
+    inventory.set(DEFAULT_VARS, "hadoop_version", HADOOP_VERSION)
+
+    inventory.write(file)
+    file.flush()
+
+
+def get_deployment_dir():
```
Comment (mbudiu-vmw) [inline bin/deploy-hdfs.py:41]: Please add a comment that this assumes that there is a deployment directory containing the ansible file etc.

**Review by mbudiu-vmw — APPROVED** [review]

File: bin/README.md (hunk 1 above)
Comment (wkk) [reply bin/README.md]: Sorry the the grammar mistake. Updated

Comment (wkk) [reply bin/README.md]: One minor issue though. This does not deploy to all the machines if there are aggregators, only webserver and workers

File: bin/deploy-hdfs.py (hunk 2 above)
Comment (wkk) [reply bin/deploy-hdfs.py]: Added a try catch block to catch import failure

File: bin/deploy-hdfs.py (hunk 3 above)
Comment (wkk) [reply bin/deploy-hdfs.py:50]: Comment added

File: bin/README.md (hunk 1 above)
Comment (mbudiu-vmw) [reply bin/README.md]: That's good

**Review by mbudiu-vmw — APPROVED** [review]

**Pull request merged**